custom-recipes/named-entity-recognition-extract/recipe.json

```diff
@@ -56,7 +46,7 @@
             "visibilityCondition": "model.ner_model=='spacy'",
             "name": "text_language_spacy",
             "label": "Language",
-            "description": "List of supported languages",
+            "description": "List of supported languages (May need to edit requirements.txt)",
```

Comment:
I don't think a plugin should reference files which are not exposed to the user in the first place. We can't expect everyone to be admin, put the plugin in dev mode, and go ham on it.

So imo either we provide an in-plugin way to do this, either we don't mention it.